lib/spdy-transport/protocol/http2/parser.js

```diff
@@ -386,6 +417,28 @@ Parser.prototype.onSettingsFrame = function onSettingsFrame(header,
       settings[name] = value;
   }
 
+  if (settings['enable_push'] !== undefined) {
```

Comment:
Another thought, it may be beneficial to introduce `._validateSettings` method, which will enclose all this validators inside of it.